example_test.go

```diff
@@ -4,20 +4,21 @@ import (
 	"fmt"
 	"github.com/DATA-DOG/godog"
 	"github.com/tranquility-bdd/tranquility"
+	"strings"
```

Comment:
My linter complains about this order of imports. I think that it should be 
`import(
        "fmt"
	"strings"

	"github.com/DATA-DOG/godog"
	"github.com/tranquility-bdd/tranquility"
)`

Reply:
@jmromanos that's odd. When I run `gofmt` and put that order of imports - it changes it back to the current order on the PR.